ios/engine/KMEI/KeymanEngine/KeymanEngine.h

```diff
@@ -1,17 +0,0 @@
-//
-//  KeymanEngine.h
-//  KeymanEngine
-//
-//  Created by Gabriel Wong on 2017-10-02.
-//  Copyright © 2017 SIL International. All rights reserved.
-//
-
-#import <UIKit/UIKit.h>
```

Comment:
This line had (effectively) automatically added an `import UIKit` for all KeymanEngine source files.  With it gone, we need to be explicit in all cases where we reference `UIKit`.